src/main/java/br/com/vivo/model/Cliente.java

```diff
@@ -0,0 +1,39 @@
+package br.com.vivo.model;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
```

Comment:
acredito que toda classe que representa uma entidade do banco tem que existir um construtor vazio.
Se for mesmo, coloca essa anotação @NoArgsConstructor em todas as classes entidade.

Reply:
O construtor vazio não é implementado por padrão? Se sim, o NoArgsConstructor teria o mesmo impacto que a existência default dele, não?